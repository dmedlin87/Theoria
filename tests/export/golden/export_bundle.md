---
title: Theo Export export-fixture
schema_version: 2024-07-01
created_at: 2024-01-01T00:00:00+00:00
export_type: search
record_count: 1
---

## Manifest
```json
{
  "export_id": "export-fixture",
  "schema_version": "2024-07-01",
  "created_at": "2024-01-01T00:00:00Z",
  "type": "search",
  "filters": {
    "query": "grace"
  },
  "totals": {
    "results": 1,
    "returned": 1
  },
  "app_git_sha": "abc1234",
  "enrichment_version": 2,
  "cursor": "passage-1",
  "next_cursor": null,
  "mode": "results"
}
```

## Records
```json
[
  {
    "kind": "result",
    "rank": 1,
    "score": 0.87,
    "document_id": "doc-1",
    "title": "Example Document",
    "snippet": "Example snippet"
  }
]
```

%% Theo export provenance manifest embedded above %%
