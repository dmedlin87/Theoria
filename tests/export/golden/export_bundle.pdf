%PDF-1.4
%
1 0 obj
<< /Type /Catalog /Pages 2 0 R >>
endobj
2 0 obj
<< /Type /Pages /Kids [3 0 R] /Count 1 >>
endobj
3 0 obj
<< /Type /Page /Parent 2 0 R /MediaBox [0 0 612 792] /Contents 4 0 R /Resources << /Font << /F1 5 0 R >> >> >>
endobj
4 0 obj
<< /Length 958 >>
stream
BT
/F1 11 Tf
14 TL
72 760 Td
(Theo Export export-fixture) Tj
T*
(Schema Version: 2024-07-01) Tj
T*
(Created: 2024-01-01T00:00:00+00:00) Tj
T*
(Export Type: search) Tj
T*
() Tj
T*
(Manifest:) Tj
T*
({) Tj
T*
(  "export_id": "export-fixture",) Tj
T*
(  "schema_version": "2024-07-01",) Tj
T*
(  "created_at": "2024-01-01T00:00:00Z",) Tj
T*
(  "type": "search",) Tj
T*
(  "filters": {) Tj
T*
(    "query": "grace") Tj
T*
(  },) Tj
T*
(  "totals": {) Tj
T*
(    "results": 1,) Tj
T*
(    "returned": 1) Tj
T*
(  },) Tj
T*
(  "app_git_sha": "abc1234",) Tj
T*
(  "enrichment_version": 2,) Tj
T*
(  "cursor": "passage-1",) Tj
T*
(  "next_cursor": null,) Tj
T*
(  "mode": "results") Tj
T*
(}) Tj
T*
() Tj
T*
(Records:) Tj
T*
([) Tj
T*
(  {) Tj
T*
(    "kind": "result",) Tj
T*
(    "rank": 1,) Tj
T*
(    "score": 0.87,) Tj
T*
(    "document_id": "doc-1",) Tj
T*
(    "title": "Example Document",) Tj
T*
(    "snippet": "Example snippet") Tj
T*
(  }) Tj
T*
(]) Tj
ET
endstream
endobj
5 0 obj
<< /Type /Font /Subtype /Type1 /BaseFont /Helvetica >>
endobj
6 0 obj
<< /Producer (Theo Exporter) /Creator (Theo Exporter) /Title (Theo Export export-fixture) /Subject (Schema 2024-07-01) /CreationDate (D:20240101000000+0000 )>>
endobj
xref
0 7
0000000000 65535 f 
0000000015 00000 n 
0000000064 00000 n 
0000000121 00000 n 
0000000247 00000 n 
0000001256 00000 n 
0000001326 00000 n 
trailer
<< /Size 7 /Root 1 0 R /Info 6 0 R >>
startxref
1501
%%EOF
