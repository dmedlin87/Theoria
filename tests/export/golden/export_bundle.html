<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <title>Theo Export export-fixture</title>
    <meta name="generator" content="Theo Exporter" />
    <meta name="theo:schema_version" content="2024-07-01" />
    <script id="theo-export-manifest" type="application/json">{
  "export_id": "export-fixture",
  "schema_version": "2024-07-01",
  "created_at": "2024-01-01T00:00:00Z",
  "type": "search",
  "filters": {
    "query": "grace"
  },
  "totals": {
    "results": 1,
    "returned": 1
  },
  "app_git_sha": "abc1234",
  "enrichment_version": 2,
  "cursor": "passage-1",
  "next_cursor": null,
  "mode": "results"
}</script>
    <script id="theo-export-records" type="application/json">[
  {
    "kind": "result",
    "rank": 1,
    "score": 0.87,
    "document_id": "doc-1",
    "title": "Example Document",
    "snippet": "Example snippet"
  }
]</script>
    <style>
      body {
        font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
        margin: 2rem;
        line-height: 1.5;
      }
      pre {
        background: #f5f5f5;
        border-radius: 0.5rem;
        padding: 1rem;
        overflow-x: auto;
      }
      h1, h2 {
        color: #1a365d;
      }
      .summary {
        margin-bottom: 2rem;
      }
      .summary dt {
        font-weight: 600;
      }
    </style>
  </head>
  <body>
    <main>
      <header>
        <h1>Theo Export export-fixture</h1>
        <dl class="summary">
          <dt>Schema version</dt>
          <dd>2024-07-01</dd>
          <dt>Created</dt>
          <dd>2024-01-01T00:00:00+00:00</dd>
          <dt>Export type</dt>
          <dd>search</dd>
        </dl>
      </header>
      <section id="manifest">
        <h2>Manifest</h2>
        <pre>{
  &quot;export_id&quot;: &quot;export-fixture&quot;,
  &quot;schema_version&quot;: &quot;2024-07-01&quot;,
  &quot;created_at&quot;: &quot;2024-01-01T00:00:00Z&quot;,
  &quot;type&quot;: &quot;search&quot;,
  &quot;filters&quot;: {
    &quot;query&quot;: &quot;grace&quot;
  },
  &quot;totals&quot;: {
    &quot;results&quot;: 1,
    &quot;returned&quot;: 1
  },
  &quot;app_git_sha&quot;: &quot;abc1234&quot;,
  &quot;enrichment_version&quot;: 2,
  &quot;cursor&quot;: &quot;passage-1&quot;,
  &quot;next_cursor&quot;: null,
  &quot;mode&quot;: &quot;results&quot;
}</pre>
      </section>
      <section id="records">
        <h2>Records</h2>
        <pre>[
  {
    &quot;kind&quot;: &quot;result&quot;,
    &quot;rank&quot;: 1,
    &quot;score&quot;: 0.87,
    &quot;document_id&quot;: &quot;doc-1&quot;,
    &quot;title&quot;: &quot;Example Document&quot;,
    &quot;snippet&quot;: &quot;Example snippet&quot;
  }
]</pre>
      </section>
    </main>
  </body>
</html>
